### PR TITLE
OEUI-63: Consume API to fetch an encounter type

### PR DESCRIPTION
**Pull request by Betty-Kebenei**
# **JIRA TICKET NAME:**

[OEUI-63: Consume API to fetch an encounter type](https://issues.openmrs.org/projects/OEUI/issues/OEUI-63)

## **SUMMARY:**
This includes consuming the API to fetch an encounter type which is needed when making a drug order.

This PR has the actions and reducers to add and fetch encounter types.

### CODE REVIEW

File: tests/actions/encounterTypes.test.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
+import expect from 'expect';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
```
Comment (geofrocker) [inline tests/actions/encounterTypes.test.js]: All the import above are globally exported in the setup file. No need to import them again

File: tests/reducers/encounterReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import expect from 'expect';
```
Comment (geofrocker) [inline tests/reducers/encounterReducer.test.js]: No need to import expect

**Comment by fuchodeveloper** [comment]
Looks good

File: app/js/actions/encounterType.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import * as types from './actionTypes';
```
Comment (dkayiwa) [inline app/js/actions/encounterType.js]: For the above line, did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

**Pull request merged**